cfgov/jinja2/v1/_includes/molecules/_vars-content.html

```diff
@@ -0,0 +1,9 @@
+{% macro _privacy_statement() %}
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/69da2061af1629219bd5b5f70e34a06e49c57ba9